edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/models/DockerStats.cs

```diff
@@ -130,14 +130,18 @@ public CpuUsage(double? total_usage, double? usage_in_kernelmode, double? usage_
     public class MemoryStats
     {
         [JsonConstructor]
-        public MemoryStats(double? limit, double? max_usage, double? usage, MemoryStatsExtended stats)
+        public MemoryStats(double? privateworkingset, double? limit, double? max_usage, double? usage, MemoryStatsExtended stats)
```

Comment:
```suggestion
        public MemoryStats(double? privateWorkingSet, double? limit, double? max_usage, double? usage, MemoryStatsExtended stats)
```

Reply:
btw what is private working set?

Reply:
this is the json constructor, the parameter name must exactly match the json it's parsing
```
        "memory_stats": {
            "commitbytes": 124702720,
            "commitpeakbytes": 128036864,
            "privateworkingset": 92790784
        },
```

Reply:
@philipktlin If you had the same question I did, I didn't know what a working set was in the context of memory. I found an answer [here](https://stackoverflow.com/questions/5405693/help-understanding-windows-memory-working-set#:~:text=Simply%20said%2C%20the%20working%20set,out%20(i.e.%20in%20RAM).&text=If%20your%20program%20uses%20more,are%20not%20necessarily%20%22gone%22.)

@lfitchett What does private imply here?

Reply:
private is reserved for that docker container's use. So if that's eh's container, its how much RAM eh is using